src/Misc/layoutbin/darwin.svc.sh.template

```diff
@@ -29,7 +35,11 @@ function failed()
 }
 
 if [ ! -f "${TEMPLATE_PATH}" ]; then
-    failed "Must run from runner root or install is corrupt"
+    if [[ $IS_CUSTOM_TEMPLATE = 0 ]]; then
+        failed "Must run from runner root or install is corrupt"
+    else
+        failed "Invalid GITHUB_ACTIONS_RUNNER_SERVICE_TEMPLATE env variable"
```

Comment:
Should we print out `GITHUB_ACTIONS_RUNNER_SERVICE_TEMPLATE` as part of the error, so the customer can check their setting?
ex:
  `failed "Fail to find '$GITHUB_ACTIONS_RUNNER_SERVICE_TEMPLATE' 
 using GITHUB_ACTIONS_RUNNER_SERVICE_TEMPLATE env variable"`